src/software/embedded/ansible/playbooks/config.yml

```diff
@@ -0,0 +1,7 @@
+# These are config for the robots and not ansible play scripts 
```

Comment:
instead of putting it default values, just put a template for people to fill out:

```
robot_id:             [Robot's robot ID]
channel_id:           [usually 0]
network_interface:    [network interface from `ifconfig`]
kick_coeff:           [ask elec]
kick_constant:        [ask elec]
chip_pulse_width:     [ask elec]
```